.github/workflows/ci_tests.yaml

```
@@ -85,9 +85,9 @@ jobs:
           - os: 'ubuntu-latest'
             python-version: '3.11'  # Can't be 3.10 or 3.12.
             numpy-version: '1.24'
-            pandas-version: ''
+            pandas-version: '=2.0'
```

Maybe we test Pandas 2.1, which is still `pandas <2.2`?

```suggestion
            pandas-version: '=2.1'
```

---

Sounds good to test three different pandas versions.